crates/stellar-scaffold-cli/tests/it/build_clients/dev.rs

```diff
@@ -9,7 +9,7 @@ async fn dev_command_watches_for_changes_and_environments_toml() {
     TestEnv::from_async("soroban-init-boilerplate", |env| async {
         Box::pin(async move {
             let mut dev_process = env
-                .stellar_scaffold_process("dev", &["--build-clients"])
+                .stellar_scaffold_process("watch", &["--build-clients"])
```

Comment:
We should probably rename this file, too

Reply:
Noted, on it